lms/views/api/exceptions.py

```diff
@@ -170,6 +171,7 @@ def http_bad_request(self):
         context=CanvasAPIPermissionError
     )
     @exception_view_config(context=Exception)
+    @exception_view_config(context=SerializableError)
```

Comment:
I wonder if we could use 

`exclog.ignore` https://docs.pylonsproject.org/projects/pyramid-exclog/en/latest/#settings

to also don't show these on the logs? They are bit noisy. 

I don't think we need to do that right now.